### PR TITLE
Cds 98303

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by SushrutHarness — APPROVED** [review]
small sugs

File: docs/continuous-delivery/verify/cv-concepts/cv_templates.md [hunk 1]
```diff
@@ -1,30 +1,57 @@
...
+       * If you want to create an environment, select **+ Add New**, enter the following information, and then select **Save**. 
+       
+          *  **Name**: Enter a name for the environment.
+          *  **Description (Optional)**
+          *  **Tag (Optional)** 
+          *  **Environment Type**: Select an environment type. The available options are **Production** and **Non-Production**.
+
+    Harness automatically populates the **Monitored Service Name** field by combining the service and environment names. This field is not editable.
+
+After creating the monitored service template, you can edit it to add a Health Source of your preference. While a monitored service entity contains a lot more than a set of health sources, the presence of health sources is sufficient for verification purposes. Hence, a monitored service template can be created only with required health sources. For more details, go to [Monitored service template documentation](/docs/service-reliability-management/monitored-service/monitored-service-template-quickstart).
```
Comment (brian-f-harness) [inline docs/continuous-delivery/verify/cv-concepts/cv_templates.md:52]:
```suggestion
After creating the monitored service template, you can edit it to add a Health Source of your preference. While a monitored service entity contains a lot more than a set of health sources, the presence of health sources is sufficient for verification purposes. Hence, a monitored service template can be created only with the required health sources. For more details, go to [Monitored service template documentation](/docs/service-reliability-management/monitored-service/monitored-service-template-quickstart).
```

**Review by brian-f-harness — APPROVED** [review]
LGTM

**Review by SushrutHarness — APPROVED** [review]
LGTM

**Pull request merged**